src/Campaigns/Blocks/CampaignComments/resources/edit.tsx

```diff
@@ -0,0 +1,96 @@
+import {InspectorControls, useBlockProps} from '@wordpress/block-editor';
+import {BlockEditProps} from '@wordpress/blocks';
+import {PanelBody, TextControl, ToggleControl} from '@wordpress/components';
+import {__} from '@wordpress/i18n';
+import CampaignComments from './shared/components/CampaignComments';
+import useCampaign from '../../shared/hooks/useCampaign';
+import {CampaignSelector} from '../../shared/components/CampaignSelector';
+import {useEffect} from 'react';
+
+export default function Edit({
+    attributes,
+    setAttributes,
+    clientId,
+}: BlockEditProps<{
+    blockId: string;
```

Comment:
Looks like you can share the `AttributeProps` you created elsewhere, maybe make a types file so both can access

Reply:
Sounds good. I pulled the CommentData out & put the types inline as well. 53bb7453150382b8aca3a71b0cfba1db5ccfeb5e